atomisticparsers/gromacs/parser.py

```diff
@@ -810,19 +810,66 @@ def get_gromacs_file(self, ext):
 
         return os.path.join(self._maindir, files[counts.index(min(counts))])
 
+    def get_all_gromacs_files(self, ext):
```

Comment:
I suggest to extend the current `get_gromacs_file` function to take a bool arg `check_if_readable`.